apps/aeoracle/test/aeoracle_SUITE.erl

```diff
@@ -462,6 +464,37 @@ query_response(Cfg, QueryOpts) ->
 
     {OracleKey, ID, S2}.
 
+query_response_fee_depends_on_response_size(Cfg) ->
+    {OracleKey, ID, S1} = query_oracle(Cfg, #{}, #{}),
+    Trees               = aeo_test_utils:trees(S1),
+    Env                 = aetx_env:tx_env(?ORACLE_RSP_HEIGHT),
+
+    SmallResponse  = <<"small_response">>,
+    BiggerResponse = <<"bigger_oracle_response">>,
+    size(SmallResponse) < size(BiggerResponse),
```

Comment:
Is this meant to be assertion?

Reply:
Yes, ☕️ made me realize it is not going to work :)